src/helpers/optionalStacking.ts

```diff
@@ -15,3 +22,66 @@ export function getViewByTwoAttributes(
     }
     return [viewBy] as VisualizationObject.IVisualizationAttribute[];
 }
+
+function getStackConfigurationForArea(config: IChartConfig = {}) {
```

Comment:
`helpers/utils` is the place for shared functionality... this functions 
`getStackConfigurationForArea, getPropsForArea` are specific only for AreaChart... which means it belongs to `AreaChartHelper/AreaChartUtils` or somewhere closer to component which is using this functions... :)